docs/ref-info/ref-integration-approaches.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Integration Approaches
+description: Reference information about the integration approaches available for UID2 integration.
+hide_table_of_contents: false
+sidebar_position: 06
+---
+
+import Link from '@docusaurus/Link';
+
+# Integration Approaches
+
+For publishers integrating UID2 into their workflows, or advertisers integrating UID2 support, there are three broad integration approaches. UID2 integrations can be implemented entirely on the client side, entirely on the server side, or partially on the client side and partially on the server side (client-server).
+
+For details, see:
+
+- [Client-side integration](#client-side-integration)
+- [Client-server integration](#client-server-integration)
+- [Server-side integration](#server-side-integration)
+
+## Client-Side Integration
+
+In a client-side integration, UID2 tokens are generated and refreshed on the client side.
+
+For example:
+
+- Publishers generate UID2 tokens on the client side for bidstream use, as well as refreshing the tokens.
+- Advertisers generate UID2 tokens on the client side for tracking pixels.
+
+Examples of documentation for publisher client-side integrations:
+
+- [UID2 Client-Side Integration Guide for Prebid.js](../guides/integration-prebid-client-side.md)
+- [Client-Side Integration Guide for JavaScript](../guides/integration-javascript-client-side.md)
+- [UID2 Client-Side Integration Guide for Mobile](../guides/integration-mobile-client-side.md)
+
+Advertisers integrating on the client side can use the JavaScript SDK:
+
+- [Client-Side Integration Guide for JavaScript](../guides/integration-javascript-client-side.md)
+
+### Client-Side Integration: Security Values
```

Comment:
We made up a term for Subscirption ID + Public Key - _Client Keypair_ but we never used this term on doc site (it exists in source code and internal doc only).

_If you choose a client-side integration, you'll get a client keypair which consists of two values that identify you to the UID2 servers: **Subscription ID** and **Public Key**._

This makes sense?

Reply:
@sunnywu it totally does. Not keen on introducing this new term in just one new doc though. it should be on the Credentials page (which, if you have an idea for a new name, LMK).
Adding it, but will reach out to you and we need to do a bit more on this.